controllers/api/post_routes.js

```diff
@@ -0,0 +1,45 @@
+const router = require("express").Router();
+const { Post } = require("../../models");
+
+// New Post rufen und in die Datebank packen.
+router.post("/", async (req, res) => {
+  // Body, da wir es mit der post.js so bestimmt haben.
+  //   try/catch handling bei Fehlerfall.
+  try {
+    const postData = req.body;
+    await Post.create({
+      title: postData.title,
```

Comment:
nice job placing the content of the body against keys here! 
It is a much safer way of creating a new post, than just passing the req.body. as anyone could send anything in the body of a request and this could potentially break your application.
For further security, you could add logic before line 8 to check that all of the required fields are present in the request. if not, you could then respond with an error that is displayed on the page asking the user to submit the correct information :-)